### ISSUE TITLE
Kolaž slik za prikaz utrinka na seznamu namesto ene same slike

### ISSUE THREAD

**Issue by bostjan** [aokranj/website#3]
Zgleda mal lame :)

**Comment by HriBB** [comment]
Na desktop size sm že popravil, na telefončkih se še vedno prikazuje velika slika. Tukej je mal problem ker folk ne bo znal pravilno vstavit galerije oziroma bo uploadal čudne fotke ... skratka en kup spremenljivk na katere nimamo vpliva in lahko pokvarijo logiko. Bom še malo premislil kako bi blo najbolš naredit

**Comment by bostjan** [comment]
A tole ti je nekdo reku "dej nazaj tko k je blo prej"? :)

**Comment by HriBB** [comment]
Ja. Več njih :) Prosim preveri na telefonu, tablici, laptopu, 4k, ... ko boš mel čas. Če karkoli ne dela sporoči, drugač lahko tud zapreš task.
